domain/bakerystorage/state/config.go

```diff
@@ -0,0 +1,142 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package state
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/canonical/sqlair"
+	"github.com/go-macaroon-bakery/macaroon-bakery/v3/bakery"
+	"github.com/juju/errors"
+
+	coredatabase "github.com/juju/juju/core/database"
+	"github.com/juju/juju/domain"
+	internaldatabase "github.com/juju/juju/internal/database"
+)
+
+type ConfigState struct {
```

Comment:
Same comment about the word `Config`.

Reply:
As above, renames to `BakeryConfigState`